src_tests/unittests/osm_configurator/model/project/test_export.py

```diff
@@ -1,46 +1,33 @@
 from __future__ import annotations
 
 import os
-import shutil
 from pathlib import Path
-from typing import TYPE_CHECKING
 
 from src.osm_configurator.model.project.active_project import ActiveProject
-from src.osm_configurator.model.project.calculation import file_deletion
 from src_tests.definitions import TEST_DIR
-
-if TYPE_CHECKING:
-    from src.osm_configurator.model.project.calculation.file_deletion import FileDeletion
-
-
-def _prepare_project_folder(path_to_new_project: Path, path_old_data: Path):
-    # Prepare result folder
-    deleter: FileDeletion = file_deletion.FileDeletion()
-    deleter.reset_folder(path_to_new_project)
-
-    # move the files from data to it
-    try:
-        shutil.copytree(path_old_data, path_to_new_project)
-    except:
-        pass
-    with open(os.path.join(path_to_new_project, "application_settings.json"), "w") as file:
-        file.write(str(path_to_new_project))
+import src.osm_configurator.model.application.application_settings as application_settings_i
 
 
 class TestExport:
```

Comment:
There are Assertions in this tests, but they are not really meaningful. It is only tested if the return value is correct. I would add an Assertion that tests whether the file was actually created